README.md

```diff
@@ -38,10 +39,6 @@ Through Github
 - Linter test in local:
   `npx stylelint "**/*.{css,scss}"`
 
-### Deployment
-
-- Not deployed yet
-
 ## Authors
```

Comment:
- [ ] OPTIONAL
I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by a sole contributor.